src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -287,7 +287,6 @@ export class CalciteRadioButton {
         this.label = document.createElement("calcite-label");
         this.label.setAttribute("dir", getElementDir(this.el));
         this.disabled && this.label.setAttribute("disabled", "");
-        this.label.setAttribute("for", `${this.guid}-input`);
```

Comment:
Removing this line because it was causing the radioButtonChange event to fire more than once when clicking the embedded `calcite-label`.  `calcite-radio-button`'s `this.onClick` handler listens for clicks to all children, of which `calcite-label` is included, so this extra `for` adds a duplicate click event because calcite-label clicks the corresponding calcite-input when the for attribute is provided.